Formula/neovim.rb

```diff
@@ -78,8 +78,8 @@ def install
   def caveats; <<-EOS.undent
       The Neovim executable is called 'nvim'. To use your existing Vim
       configuration:
-          ln -s ~/.vimrc ~/.nvimrc
-          ln -s ~/.vim ~/.nvim
+          ln -s ~/.vimrc ~/.config/nvim/init.vim
```

Comment:
Yeah that makes sense. I created [a new PR for this change](https://github.com/neovim/homebrew-neovim/pull/105), since this one has already been merged and I wasn't sure if I could or should re-open it.